Initial draft for travel sponsorship

castrojo opened the pull request:
This is a PR of Paris' draft. I've added TODOs in the sections that need it, and did some minor edits. Some things steering might also want to consider:

- We should consider revising SIG/committee/WG/UG governance to include seeding and recommending people for this program as we shouldn't expect individuals to just find it, SIGs should know that they should be on the lookout for people who might need this program. 

- The equity section is written to ensure new people are sponsored, perhaps also explictly consider people who are repeat high-performers who show consistent contributions? 


I will be pushing editing updates as steering gives feedback so I'll be pushing little fixes often. 

/hold

castrojo commented:
We'll also need a decision on where the repo will live, what exactly $certain_group is expected to be, and I know CNCF has been doing Visa letters in the past so we likely need a section mentioning how we'll liase with CNCF.

castrojo commented:
TODO: We'll also need to make sure that we send people to event registration and https://events.linuxfoundation.org/travel-visa-request/ after, venn diagram should be the same as the diversity scholarship.

justaugustus commented:
(Added an issue to track this on the SIG Release side as well: https://github.com/kubernetes/sig-release/issues/734.)

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+## Requirements
+
+ - Candidates must be a [member of the Kubernetes GitHub organization][membership] or working towards it. Exceptions: TODO
+ - Those requesting funds must indicate that they do not have funding available from another source, such as an employer or the event itself that might cover costs for presenters.
```
@castrojo @parispittman Can they request partial funding? (just one aspect of the travel or lodging)

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+## Process
+
+Submit an issue to a Kubernetes Travel Fund Issue-Only repository. 
+Your issue will go into a private repository where only TODO: $certain_group can see your information due to privacy.
```
repo is public, but issues are not-public?

castrojo replied:
Ok talked to Paris, let's just do a form, more lightweight, less overhead. Pushed a new rev.

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+Submit an issue to a Kubernetes Travel Fund Issue-Only repository. 
+Your issue will go into a private repository where only TODO: $certain_group can see your information due to privacy.
+
+Requests must be made three months before the event. Requests submitted after that will not be approved.
```
Can we say that the travel fund coordinators will come up with a deadline that is approximately three months before the event. And that the requests should be submitted before the dead line. (give some wiggle room!)

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+ - The name of the event you plan to attend.
+ - The location, dates of the event and where you will be traveling from.
+ - The presentation you intend to give, if applicable.
+ - The size of the stipend you wish to receive.
```
What would the stipend cover? (if partial)?

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+
+An appointed treasurer from the TODO: $certain_group will liaise with a member of CNCF on a regular basis to review the status of the travel fund. 
+In the event that any pending requests might not be approved because of available funds, a separate issue will be raised. 
+Generally speaking, however, members shouldn't need to worry about the status of the fund itself.
```
agreed.

dims commented on travel-sponsorship.md:
```diff
@@ -0,0 +1,82 @@
+#### Impact
+
+Preference is given to speakers, panelists, and those volunteering onsite at the conference.
+How much you contribute to the project.
```
Do we ask them to write up how they contribute?

castrojo replied:
Yeah and I think we ask someone who has attended previously to do one for us so we have an example of what a good one looks like as well.

castrojo commented:
Ok I added some clarification about the points you brought up. I figured the stipend should be used for food and expenses, not personal spending money so I added that in there. 

@parispittman  I'm not sure how the private issues gh repo would work, can public accounts file issues on a private repo? I suspect this might end up being a form.

dims commented:
/assign @dims 
/hold cancel

dankohn commented:
CNCF will be making a number of proposed changes this week to enable an integrated process for applying for diversity scholarships and travel funding. Could you please hold off on your review.

timothysc commented:
/assign @timothysc

dims commented:
@dankohn @Wendi-West Ping!  Any updates on this? Also please see Paris' request above.

nikhita commented:
@dankohn @Wendi-West :wave: Hi! Since KubeCon Amsterdam isn't too far away from KubeCon San Diego and we'll also have major holidays coming up, we'd want to have the travel sponsorship in place soon-ish. We *definitely* don't want to miss out on this for Amsterdam. :)

Can you please let us know if there are any updates or the expected timeline for it? Thanks!

idvoretskyi commented:
/cc

saiyam1814 commented:
> @dankohn @Wendi-West 👋 Hi! Since KubeCon Amsterdam isn't too far away from KubeCon San Diego and we'll also have major holidays coming up, we'd want to have the travel sponsorship in place soon-ish. We _definitely_ don't want to miss out on this for Amsterdam. :)
> 
> Can you please let us know if there are any updates or the expected timeline for it? Thanks!

is it different form the Diversity Scholarship ? If yes then it would be awesome for KubeCon Amsterdam.

idvoretskyi commented:
> is it different form the Diversity Scholarship ?

Yes. More details are coming soon.

justaugustus commented:
Any update on this?

idvoretskyi commented:
@justaugustus we are at the final round of approval, please expect an update from me by EOD today.

idvoretskyi commented:
Please see the travel policy that CNCF defined - https://github.com/cncf/foundation/blob/master/travel_funding.md.

Also, we've updated the event's website with the diversity scholarship page for KC+CNC EU 2020:
 https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/attend/diversity-scholarships/, and KC+CNC EU 2020 application on survey monkey:
https://www.surveymonkey.com/r/FPXKRJ3

parispittman commented:
@idvoretskyi thank you for working on this. 
who is the review committee for not selected contributor candidates? / how are they formed?

idvoretskyi commented:
@parispittman it's not formed yet for Amsterdam, I'll come back with more details later.

mrbobbytables commented:
@idvoretskyi any update on this? I know it won't make it for Amsterdam, but do you think we'll have in place for Shanghai / Boston?

idvoretskyi commented:
The diversity scholarship review committee has been compromised of KubeCon co-chairs, program committee members and other individuals that have been referred to us by the co-chairs/program committee. Most people sign up for the full year and commit to all three KubeCons.

nikhita commented:
@idvoretskyi just want to follow up and check if there have been any updates from the CNCF side on this?

idvoretskyi commented:
@nikhita posted on [Feb 9](https://github.com/kubernetes/steering/pull/114#issuecomment-583824449), anything else?

idvoretskyi commented:
To reiterate - CNCF has published the travel funding policy in December (https://github.com/cncf/foundation/blob/master/travel_funding.md).

Specifically, the paragraph 7(b) states - _Any applicants not selected for a Diversity Scholarship, who selected Kubernetes Contributor (volunteer, speaker, contributor) or a CNCF-organized Day Zero Co-Lo will then be sent to respective review committees for funding consideration._

dims commented:
@idvoretskyi which review committee is responsible for "Kubernetes Contributor (volunteer, speaker, contributor)" bucket?

idvoretskyi commented:
@dims please see my comment above - https://github.com/kubernetes/steering/pull/114#issuecomment-583824449.

_The diversity scholarship review committee has been comprised of KubeCon co-chairs, program committee members and other individuals that have been referred to us by the co-chairs/program committee. Most people sign up for the full year and commit to all three KubeCons._

dims commented:
@idvoretskyi ah so there's only one committee that takes care of all the buckets. i got confused as it says "respective review committees"

nikhita commented:
From the public steering meeting on May 5th -- @idvoretskyi to follow up with @dims and SC on next actionable steps

nikhita commented:
Update - @dims has sent an email to CNCF to ask more details about this. Hasn't received a response yet, but we'll follow up with this later.

Discussed in the 1 June 2020 meeting to close this PR and move any future discussions to https://github.com/kubernetes/steering/issues/109

/close
